.circleci/ci/src/pipelines/tests/pipeline-publish-docker-images.spec.ts

```diff
@@ -15,6 +15,7 @@ describe('Publish docker images tests', () => {
       buildNum: '1234',
       buildId: '1234',
       graviteeioVersion: '4.2.0',
+      apimVersionPath: './src/../../../pom.xml',
```

Comment:
Tests will be wrong after each release, won't they ?

Reply:
in this line:
`command: "sed -i 's/\"version\": \".*\"/\"version\": \"4.2.0-SNAPSHOT\"/' << parameters.apim-ui-project >>/build.json"`

Reply:
done